spec/filters/kv_spec.rb

```diff
@@ -107,6 +107,21 @@
         end
       end
     end
+    
+    context "invalid message" do
+      context "without splitter" do
+        let(:tag_1) { "_failure_tag_1" }
+        let(:tag_2) { "_failure_tag_2" }
+        let(:message) { "random text foo bar baz" }
+        let(:options) { {"tag_on_failure" => ["#{tag_1}", "#{tag_2}"]} }
+
+        it "should short circuit" do
+          subject.filter(event)
+          expect(event['tags']).to include("_failure_tag_1")
+          expect(event['tags']).to include("_failure_tag_2")
```

Comment:
given that you defined `tag_1` and `tag_2` in let blocks, you should consider using them in lines 120, 121

Reply:
Thank you for pointing that out, definitely an oversight on my part.  I'll try to rebase my branch and push these changes within the next day.